projects/js-packages/shared-extension-utils/package.json

```diff
@@ -17,6 +17,7 @@
 	},
 	"devDependencies": {
 		"jetpack-js-test-runner": "workspace:*",
+		"lodash": "4.17.21",
```

Comment:
Similarly to the last PR, maybe we could get rid of the dependency and instead rely on optional chaining and nullish coalescing?